exonum_launcher/launcher.py

```diff
@@ -190,6 +192,36 @@ def wait_for_unload(self) -> None:
         except NotCommittedError as error:
             self.launch_state.unload_status = ActionResult.Fail, str(error)
 
+    def migrate_all(self) -> None:
+        """Migrates all services from the provided config."""
+        for service_name, artifact in self.config.migrations.items():
+            migration_request = self._supervisor.create_migration_request(service_name, artifact)
+            txs = self._supervisor.send_migration_request(migration_request)
+            self.launch_state.add_pending_migration((service_name, artifact), txs)
+
+    def wait_for_migration(self) -> None:
+        """Waits for all migrations to be completed."""
+        pending_migrations = self.launch_state.pending_migrations()
+        for tx_hashes in pending_migrations.values():
+            self._explorer.wait_for_txs(tx_hashes)
+
+        for (service_name, artifact), _ in pending_migrations.items():
```

Comment:
Good catch but `pylint` doesn't pass `for service_name, artifact in pending_migrations:`. I don't know actually why. Maybe because a key type is tuple.